src/lib/y2storage/storage_manager.rb

```diff
@@ -180,9 +181,9 @@ def probe(probe_callbacks: nil)
     #
     # @raise [Storage::Exception, Yast::AbortException] when probe fails
     #
-    # @param probe_callbacks [Callbacks::Probe, nil]
-    def probe!(probe_callbacks: nil)
-      probe_callbacks ||= Callbacks::Probe.new
+    # @param callbacks [Callbacks::Probe, nil]
+    def probe!(callbacks = nil)
```

Comment:
Maybe we could take the opportunity for moving `Yast::Pkg.SourceReleaseAll if Yast::Mode.installation` to the callback.